test/IDE/complete_tf_198.swift

```diff
@@ -0,0 +1,7 @@
+// https://bugs.swift.org/browse/TF-198: `@dynamicCallable` REPL completer crash.
+// RUN: %target-swift-ide-test -repl-code-completion -source-filename=%s
+
+// TODO(TF-214): Require `python` lit feature, after it is created.
```

Comment:
Could this test construct and use its own dynamicCallable, so that it doesn't need to depend on python?

Reply:
Yes - though I think we want to be confident that things work specifically with `PythonObject`.
I'd like to leave the test as is for now, if you don't mind.

Reply:
don't mind!